docs/rfcs/0000-new-pipeline.md

```diff
@@ -15,7 +15,7 @@ Introducing a new Pipeline syntax for supporting:
 
 ```json
 {
-  "specVersion": "2.0",
+  "version": "2.0",
```

Comment:
The `version` is reversed for a pipeline itself.

Reply:
OK, rollbacked.